server/datastore/mysql/apple_mdm.go

```diff
@@ -1642,6 +1642,36 @@ func (ds *Datastore) GetMDMAppleBootstrapPackageBytes(ctx context.Context, token
 	return &bp, nil
 }
 
+func (ds *Datastore) GetMDMAppleBootstrapPackageSummary(ctx context.Context, teamID uint) (fleet.MDMAppleHostStatusSummary, error) {
+	stmt := `
+          SELECT
+              COUNT(IF(ncr.status = 'Acknowledged', 1, NULL)) AS applied,
+              COUNT(IF(ncr.status = 'Error', 1, NULL)) AS failed,
+              COUNT(IF(ncr.status IS NULL, 1, NULL)) AS pending
```

Comment:
Should all other statuses fall into this category? E.g. I think it can be `NotNow` under some circumstances?